### PR TITLE
TST/CLN: Remove makeTime methods

### PR DESCRIPTION
**Pull request by mroeschke**

### CODE REVIEW

File: pandas/tests/io/pytables/test_select.py [hunk 1]
```diff
@@ -388,7 +408,7 @@ def test_select_iterator(tmp_path, setup_path):
         # full selection
         expected = store.select_as_multiple(["df1", "df2"], selector="df1")
         results = list(
-            store.select_as_multiple(["df1", "df2"], selector="df1", chunksize=150)
+            store.select_as_multiple(["df1", "df2"], selector="df1", chunksize=2)
```
Comment (WillAyd) [inline pandas/tests/io/pytables/test_select.py:411]: Is this just for performance?

**Review by WillAyd — APPROVED** [review]
lgtm

File: pandas/tests/io/pytables/test_select.py (hunk 1 above)
Comment (mroeschke) [reply pandas/tests/io/pytables/test_select.py:411]: Yup, exactly

**Comment by amotl** [comment]
Hi. To all people finding this: In a similar spirit like [verlib2](https://pypi.org/project/verlib2/), we rescued the corresponding testing helper methods and made them available on behalf of the [pueblo](https://pypi.org/project/pueblo/) package, for all downstream packages which absolutely need them.

- https://github.com/pyveci/pueblo/pull/59

**Pull request merged**